packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/plugin-ledger-connector-fabric.ts

```diff
@@ -243,6 +284,12 @@ export class PluginLedgerConnectorFabric
         functionOutput: outUtf8,
       };
       this.log.debug(`transact() response: %o`, res);
+      const endTransactionTime = new Date();
+      this.prometheusExporter.addCurrentTransaction(
```

Comment:
@jagpreetsinghsasan If initially we only support the count of total transactions (which I agree with) then there's no need for us to store the start and end times. We could keep them anyway, but then the code is a bit harder to read for those who are trying to see how it works because the start and end transaction times are not actually being used by the exporter. The other thing is that if we leave them in, then we also have to come up with some way to clear them out of memory over time to avoid memory leaks => with the current format if I read it right it just keeps pumping the array with new objects into infinity, so it's easier (since we only return totalTxCount anyway) to just not create nor store any new objects and instead just increment a single variable that stores the tx count.

Reply:
Fixed